profiles/gcc-10-aix-72

```diff
@@ -0,0 +1,40 @@
+include(boost-options)
+include(options-libtiff)
+[settings]
+os=AIX
+os.version=7.2
+compiler=gcc
+compiler.version=10.3
+compiler.libcxx=libstdc++
+build_type=Release
+[options]
+# Fix up building b2 on AIX. This will skip toolset autodetection and make b2
+# use the CC and CXX variables for building itself (and maybe even for building
+# Boost), instead of trying to autodetect a compiler, which finds some other
+# compiler and fails.
+# See the b2 Conan recipe for discussion.
+# New b2 build script in 4.7.1 needs cxx toolset to work with use_cxx_env.
+b2:toolset=cxx
+b2:use_cxx_env=True
+[env]
```

Comment:
env is a conan1 thing - so you can just take this out

Reply:
deleted